requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/astropy/astropy.git#egg=astropy
+git+https://github.com/spacetelescope/asdf.git#egg=asdf
+git+https://github.com/spacetelescope/gwcs.git#egg=gwcs
+pytest==3.8.2
```

Comment:
This `pytest==3.8.2` can be removed.  The normal `pytest` dependency in `setup.py` is fine for dev.  The `3.8.2` peg was there to avoid `3.9.x` which was completely broken.